src/messages/IPostMessageSentHandler.ts

```diff
@@ -1,4 +1,4 @@
-import { IExecutionResult, IHttp, IRead } from '../accessors';
+import { IExecutionResult, IHttp, IModify, IRead } from '../accessors';
```

Comment:
We have talked a little about how Rocketlets are going to be distributed and the general gist is that we are going to just go off the description that is provided in the `rocketlet.json` file for displaying what the Rocketlet will do. This is open for people to be detailed or not about what their Rocketlet does, however it means less work for displaying a huge list of what the Rocketlet does by what their code implements. Along with this fact is that the team has suggested we don't force Typescript usage, due to people rejecting using Rocketlets because of Typescript, and as a result of that we can not assume that the source code given to us will implement interfaces since regular Javascript doesn't support it and we are going to have to fall back on determining which methods they actually implement.

So, all in all, I think that Rocketlets should be flexible still and not over-engineered so that it is easy for people to get started...in general allow people to do what they want and how, but heavily suggest how they do it by the framework. I still think we should force them to not modify the message in after it's been sent for the user's experience, since once something is sent and then less than a second later it gets updated "automagically" it won't feel natural. This is the reason why I went with the naming schema when I updated the builder interface yesterday.